cfpb/cfgov-refresh
# add container for better whitespace around at capacity message

Fixes GHE/CFGOV/data-research/issues/438

There is no space between the paragraph text and the footer when the 'at capacity' message loads on the conference page. This is especially noticeable when the "SECONDARY NAV EXCLUDE SIBLING PAGES" checkbox is checked so the sidebar content is shorter than the page content.

## Additions

- edited the html template for the conference registration form so that a `block` div container wraps around the full-width-text organism output. This occurs by default in the page template for the full-width-text element on browse pages, so it mimics the layout expected in most full-width-text contexts. 


## Testing

- check out locally
- Create a 'conference registration form' element in the Wagtail admin
- set "Capacity" field to `0`
- Add some text to the "At capacity message" field
- View page and see that the `block block__flush-top` div element wraps around the text and there is nice pretty whitespace below it

## Review

- @chosak 
- @nataliafitzgerald 

## Screenshots

### Before
![screen shot 2016-11-18 at 4 39 48 pm](https://cloud.githubusercontent.com/assets/702526/20448450/06bedefa-adb2-11e6-9149-d7baf0da2e48.png)


### After
![screen shot 2016-11-18 at 4 50 25 pm](https://cloud.githubusercontent.com/assets/702526/20448443/fda2e50a-adb1-11e6-808d-31f1f31e4df7.png)


### After with inspector
![screen shot 2016-11-18 at 5 05 12 pm](https://cloud.githubusercontent.com/assets/702526/20448442/f8f77e3a-adb1-11e6-8775-e0ef67d95e00.png)


## Checklist
